src/main/scala/com/databricks/labs/overwatch/pipeline/BronzeTransforms.scala

```diff
@@ -1144,4 +1044,156 @@ trait BronzeTransforms extends SparkSessionWrapper {
     cleanDF
   }
 
+  def getAuditLogsDfFromSystemTables(
+                               fromTime: LocalDateTime,
+                               untilTime: LocalDateTime,
+                               organizationId: String,
+                               auditLogConfig: AuditLogConfig
+                             ): DataFrame = {
+    try {
+      println("Fetching data from system.access.audit")
```

Comment:
If you're choosing to keep this in STDOUT, then also add the organization_id so that we can tell what's happening in the job's output

Reply:
done